webui/react/src/components/NavigationSideBar.tsx

```diff
@@ -256,13 +257,17 @@ const NavigationSideBar: React.FC = () => {
           {dashboardEnabled ? null : (
             <section className={css.launch}>
               <div className={css.launchBlock}>
-                <Button className={css.launchButton} onClick={() => openJupyterLabModal()}>
-                  Launch JupyterLab
-                </Button>
-                {settings.navbarCollapsed ? (
-                  <Button className={css.launchIcon} onClick={() => openJupyterLabModal()}>
-                    <Icon name="jupyter-lab" />
+                <div className={css.launchButton}>
```

Comment:
I actually like the idea of a `fullWidth` button the most. I was not sure if we did not include it in the original UI Kit because we wanted all buttons to be solely based off of the size of the content inside. But if a full width button is okay then let's do that. 

I will update this PR with a full width button implementation.

Reply:
Nice work implementing that, thanks!